internal/tss/services/transaction_manager_test.go

```diff
@@ -182,14 +173,11 @@ func TestBuildAndSubmitTransaction(t *testing.T) {
 		assert.Equal(t, xdr.TransactionResultCodeTxTooLate, resp.Code.TxResultCode)
 		assert.Empty(t, err)
 
-		var txStatus string
-		err = dbConnectionPool.GetContext(context.Background(), &txStatus, `SELECT current_status FROM tss_transactions WHERE transaction_hash = $1`, payload.TransactionHash)
-		require.NoError(t, err)
-		assert.Equal(t, string(entities.ErrorStatus), txStatus)
+		tx, _ := store.GetTransaction(context.Background(), payload.TransactionHash)
```

Comment:
Can we assert the error for **all tests**? We may have a case where it returns a value and error.

Reply:
I did not assert the error this particular call, because as I stated above, this is not a unit test for GetTransaction. But I did assert the errors and the response for every BuildAndSubmitTransaction call in these tests